### ISSUE TITLE
ftgen with array parameter does not work in csoundweb

### ISSUE THREAD

**Issue by docb** [csound/csound#1475]
```
sr=44100
ksmps=32
0dbfs=1
nchnls=2
  givals[] fillarray 0,1024,0.5,1024,0.3,1024,-0.6,1024,0
  gitab1 ftgen 0,0,4096,7,0,1024,0.5,1024,0.3,1024,-0.6,1024,0
  gitab ftgen 0,0,4096,8,0,1024,0.5,1024,0.3,1024,-0.6,1024,0
  gitab2 ftgen 0,0,4096,7,givals
  gitab3 ftgen 0,0,4096,8,givals
instr 1 
  
endin
```
this code produces 4 korrekt tables on the commandline but in csound-web:
ftable 101:	4096 points, scalemax 1.000
[...]
ftable 102:	4096 points, scalemax 1.000
[...]
**ftable 103:	4096 points, scalemax 0.000
ftable 104: illegal x interval**
f104     0.00  4096.00     8.00    0.00 ...
INIT ERROR in instr 0 line 9: ftgen error
 from file project.orc (2)
 from file project.csd (1)
gitab3	ftgen	0	0	4096	8	givals	
header init errors
Failed to compile CSD file.

**Comment by vlazzarini** [comment]
Is the web version the same as the desktop? I can't see why there would be a difference if they both use the same source code. @kunstmusik, @hlolli any ideas?

**Comment by docb** [comment]
this i wanted to check but run into issue #1476

**Comment by kunstmusik** [comment]
All of the Webaudio Csound builds are always built with same source as desktop and the last official release was 6.15.0. 

I tested here and found that I could reproduce the error in Csound Web-IDE, but that has older version of Csound (6.13.0). I think the problem is not in ftgen with arrays but this part here:

ftable 104: illegal x interval
f104 0.00 4096.00 8.00 0.00 ...

which makes me think it is related to something that changed with GEN08 over time.  I can run the code with live.csound.com which is using 6.16.0-beta4. 

I would suggest that this is just Web-IDE related not csound-specific and should be moved over to https://github.com/csound/web-ide/issues for tracking. The issue should be addressed once we can finish the 6.16.0 release and update the version of csound used in the web-ide.
